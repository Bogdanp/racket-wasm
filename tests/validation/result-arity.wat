(module
 (memory 1 10)
 (func
  (result i32 i32)
  (i32.const 10)))
