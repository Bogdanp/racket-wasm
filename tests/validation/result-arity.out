(#f . "result arity error\n  expected: [i32 i32]\n  found: [i32]\n  at: (funcidx 0)")